packages/modules/viewer-react/src/components/error/ErrorBoundary.tsx

```diff
@@ -0,0 +1,43 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ErrorPage } from "@itwin/itwinui-react";
+import type { PropsWithChildren, ReactNode } from "react";
+import React, { Component } from "react";
+
+interface State {
+  fallback: boolean;
+  error: Error;
+}
+
+export class ErrorBoundary extends Component<
+  PropsWithChildren<ReactNode>,
+  State
+> {
+  override state: State = {
+    fallback: false,
+    error: new Error(),
+  };
+
+  static getDerivedStateFromError(e: Error): State {
```

Comment:
nice job using the state from above instead of redefining it inline :)

Reply:
Thanks!